terraform/cloudwatch.tf

```
@@ -0,0 +1,15 @@
+resource "aws_cloudwatch_log_group" "nessus_data" {
+  name = "/gds/nessus-data"
+
+  tags = {
+    Environment = "production"
+    Application = "Nessus scans"
+  }
+}
+
+resource "aws_cloudwatch_log_subscription_filter" "log_subscription" {
+  name            = "log_subscription"
+  log_group_name  = "/gds/nessus-data"
```

```suggestion
  log_group_name  = "/gds/nessus-scans"
```
Could we call this `scans` instead of the generic `data`?